exp_build/templates/partials/content.handlebars

```diff
@@ -12,6 +12,9 @@
   {{#if isText}}
     {{{content}}}
   {{/if}}
+  {{#if isComponent}}
```

Comment:
If we only check for isComponent and include code editor, then we will not be able to add more components in future. Please design it in a future-proof manner